deliver/lib/deliver/options.rb

```diff
@@ -6,6 +6,7 @@ class Options
     def self.available_options
       user = CredentialsManager::AppfileConfig.try_fetch_value(:itunes_connect_id)
       user ||= CredentialsManager::AppfileConfig.try_fetch_value(:apple_id)
+      user ||= ENV["DELIVER_USER"]
```

Comment:
DELIVER_USER is referenced in [credentials manager.](https://github.com/fastlane/fastlane/blob/master/credentials_manager/lib/credentials_manager/appfile_config.rb#L76) Maybe we need to add `DELIVER_USERNAME` to there as well?